Common/DCAFitter/GPU/hip/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+# All rights not expressly granted are reserved.
+#
+# This software is distributed under the terms of the GNU General Public
+# License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+set(CMAKE_HIP_FLAGS "${CMAKE_HIP_FLAGS} -fgpu-rdc")
+o2_add_hipified_library(DCAFitterHIP
+                        SOURCES ../cuda/DCAFitterN.cu
+                        PUBLIC_INCLUDE_DIRECTORIES ../../include
+                        PUBLIC_LINK_LIBRARIES O2::MathUtils
+                                              O2::ReconstructionDataFormats
+                                              O2::DetectorsBase
+                                              hip::host
+                        PRIVATE_LINK_LIBRARIES O2::GPUTrackingHIPExternalProvider
+                        TARGETVARNAME targetNAme)
+# target_compile_options(${targetName} PRIVATE $<$<COMPILE_LANGUAGE:HIP>:-fgpu-rdc>)
+# target_link_options(${targetName} PRIVATE $<$<COMPILE_LANGUAGE:HIP>:-fgpu-rdc>)
+
+# o2_add_hipified_library(DCAFitterAPIHIP
+#                         SOURCES ../cuda/DCAFitterGPUAPI.cu
+#                         ../cuda/DCAFitterN.cu
+#                         PUBLIC_INCLUDE_DIRECTORIES ${CMAKE_SOURCE_DIR}/Common/DCAFitter/GPU
+#                                                    ../cuda
+#                         PUBLIC_LINK_LIBRARIES O2::GPUCommon
+#                                               O2::ReconstructionDataFormats
+#                         PRIVATE_LINK_LIBRARIES O2::DCAFitterHIP O2::GPUTrackingHIPExternalProvider
+#                         TARGETVARNAME targetName)
```

Comment:
Uncomment this part (and possibly the inclusion of `O2::GPUTrackingHIPExternalProvider`) to trigger the failure but possibly pointing to the real issue